core/src/main/java/org/kiwix/kiwixmobile/core/main/KiwixTextToSpeech.kt

```diff
@@ -84,7 +84,7 @@ class KiwixTextToSpeech internal constructor(
    * Reads the currently selected text in the WebView.
    */
   fun readSelection(webView: WebView) {
-    webView.loadUrl("javascript:tts.speakAloud(window.getSelection().toString());", null)
+    webView.loadUrl("javascript:tts.speakAloud(window.getSelection().toString());")
```

Comment:
Why was this change required?

Reply:
Android 11  handles null differently. It doesn't compile if we pass the null value in `additionalHttpHeaders`.
Since we don't need `additionalHttpHeaders` we can remove them
The below error shows if we pass null value
```
Null can not be a value of a non-null type (Mutable)Map<String!, String!>
```

Reply:
According to android
>If you are writing in Kotlin, when upgrading from the Android 10 to the Android 11 SDK, you may notice that there are some new compiler warnings and that previous warnings may have been upgraded to errors. This is intended and a feature of the Kotlin compiler — these warnings tell you that you may be writing code that crashes your app at runtime (a risk you would miss entirely if you weren’t writing in Kotlin). As you encounter these warnings and errors, you can handle them by adding null checks to your code.